package.json

```diff
@@ -11,8 +11,8 @@
     "exclude": [
       "**/**/**/swaggerDefinition.js",
       "**/**/**/index.js",
-      "**/**/**/sendResult.js",
-      "**/**/**/validator.js"
```

Comment:
great job @urbanishimwe, however, avoid irrelevantly changing files because I see you just changed the order nothing much.